README.md

```diff
@@ -131,7 +131,7 @@ obt init o-my-new-component
 ### Migrating from 7.X.X to 8.X.X
 
 
-OBT no longer supports NodeJS v6 because it uses async functions. To use this version of OBT, you will need NodeJS v8 or above.
+OBT no longer supports NodeJS v6 because it uses async functions. To use this version of OBT, you will need NodeJS v10 or above.
```

Comment:
This should be under 8.xx to 9.xx as we are already on version 8

Reply:
this was a mistake I need to write a new migration guide, thanks :)